rootfs/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.3
+
+RUN apk add --update-cache curl bash openssl sudo python3 && rm -rf /var/cache/apk/*
+RUN ln -s /usr/bin/python3 /usr/bin/python
+RUN curl -sSL https://raw.githubusercontent.com/pypa/pip/7.1.2/contrib/get-pip.py | python -
```

Comment:
Are you sure it's not just complaining about "a newer version is available?" No worries--leaving it at 7.1.2 is fine if the current version is problematic.